### PR TITLE
[#1478] landing visual tweaks

### PR DESCRIPTION
**Pull request by aluarius**
## Description
close: #1478 

All items from 1478 were closed, but i should mention some moments:

> set primary color to 2550c0 (not 4382d9)

There is no 4382d9 at all, but i can see 2550c0. So, i guess, it was already changed.

> docs menu items should lead to introduction > Overview (not to the roadmap)

Done, but it seems strange, that overview & doc links lead to the one page. May be overview link should have '/' href? It's logically, cause of another item: "overview should be black on the landing page".

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/components/header/components/LanguageDropdown.vue [hunk 1]
```diff
@@ -78,11 +78,11 @@ export default defineComponent({
 
 <style lang="scss">
 .language-dropdown {
-  font-weight: 700;
+  font-weight: 600;
```
Comment (Derranion) [inline packages/docs/src/components/header/components/LanguageDropdown.vue:81]: Button style will overwrite this, that's why we could just rely on `.va-button__content` and remove this, thanks.

**Review by Derranion — APPROVED** [review]
Amazing, thanks a lot for your help, we are truly appreciate it.

Default primary color is set here:
```
  [ThemeName.DEFAULT]: {
    primary: '#2C82E0',
```

Should be fine it is already pretty close to `#4382D9`.

I've assigned you to the related task and checked all accomplished changes.

There is only one left:
- [ ]  increase the default p font size to 1.2rem (not 1.1 rem)

Please tell us if you want us to finish this.

File: packages/docs/src/components/header/components/LanguageDropdown.vue (hunk 1 above)
Comment (aluarius) [reply packages/docs/src/components/header/components/LanguageDropdown.vue:81]: > * [ ]   increase the default p font size to 1.2rem (not 1.1 rem)
> 
> Please tell us if you want us to finish this.

Hm, i thought i closed this item, will check & finish at monday.

Comment (aluarius) [reply packages/docs/src/components/header/components/LanguageDropdown.vue]: Default p font-size was already increased, unnessesary font-weight was removed.

**Comment by Derranion** [comment]
Well done.

On hold - waiting for Nuxt update.

**Comment by Derranion** [comment]
@aluarius Hey,

Could you check this comment?
Requires minor update but it is lower priority than Nuxt update, could be done later.
https://github.com/epicmaxco/vuestic-ui/issues/1478#issuecomment-1101652954

**Comment by aluarius** [comment]
Asked by Maxim to merge it.

**Pull request merged**